README.rst

```diff
@@ -80,6 +80,23 @@ For questions about how to use Cirq post to
 `Quantum Computing Stack Exchange <https://quantumcomputing.stackexchange.com/>`__ with the
 `cirq <https://quantumcomputing.stackexchange.com/questions/tagged/cirq>`__ tag.
 
+How to cite Cirq
+----------------
+
+Cirq is uploaded to Zenodo automatically. Click on this badge [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4062499.svg)](https://doi.org/10.5281/zenodo.4062499) to see all the citation formats for all versions.
+
+An equivalent BibTex format reference is below for all the versions:
+
+@software{quantum_ai_team_and_collaborators_2020_4062499,
+  author       = {Quantum AI team and collaborators},
+  title        = {Cirq},
+  month        = Aug,
```

Comment:
Why August? The URL on line 97 points to release from October 2.

Reply:
Copy paste error! Fixed, thank you!